core/php8.2Action/Dockerfile

```diff
@@ -0,0 +1,93 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# build go proxy from source
+ARG GO_PROXY_BASE_IMAGE=golang:1.18
```

Comment:
This go version is end of life as of 01 Feb 2023, can we bump this? I think go _should_ be mostly backwards compatible in this case.

Reply:
This is controlled "globally" through https://github.internal.digitalocean.com/serverless/main/blob/19a747e300dfdc6a5f6a4fde7b8a811e664cd2b2/images/buildRuntimes.sh#L29, so we're running this on golang:1.20 actually. Since this is a new file though, we might as well fix it here as well.